### PR TITLE
dvmkv2mp4 on MacOS

### PR DESCRIPTION
**Pull request by trytodev**
Hi! Not sure if this PR is really needed, but I've managed to launch dvmkv2mp4 on MacOS by changing two rows in exec file and using ionice analog for mac. This small comment might help somebody who wants to use dvmkv2mp4 on apple system.

### CODE REVIEW

**Comment by gacopl** [comment]
Well thanks for testing but do it right like a man :D

https://remarkablemark.org/blog/2020/10/31/bash-check-mac/

**Comment by trytodev** [comment]
Hello! I'm finally got some time to check the link. Done and tested on few small .mkv files. Also removed second set of "ionc" variable, everything should be ok with first entrance in the beginning.

File: README.md [hunk 1]
```diff
@@ -71,6 +71,18 @@ wget https://raw.githubusercontent.com/gacopl/dvmkv2mp4/main/dvmkv2mp4
 chmod a+x dvmkv2mp4
 mv dvmkv2mp4 /usr/local/bin/
 ```
+
+To use dvmkv2mp4 in OS X:
+- download and install ionice-MacOS 
+- update dvmkv2mp4 by changing
```
Comment (gacopl) [inline README.md:77]: Please update this comment and I'll merge many thanks!

**Review by gacopl — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (trytodev) [reply README.md]: Comment updated

**Pull request merged**